extension/src/hyperloglog.rs

```diff
@@ -38,13 +38,29 @@ pub fn hyperloglog_trans(
     fc: pg_sys::FunctionCallInfo,
 ) -> Option<Internal> {
     // let state: Internal = Internal::from_datum();
-    hyperloglog_trans_inner(unsafe{ state.to_inner() }, size, value, fc).internal()
+    hyperloglog_trans_inner(unsafe{ state.to_inner() }, size, value, fc,unsafe {pgx::get_getarg_type(fc, 2)}).internal()
 }
+
+const APPROX_COUNT_DISTINCT_DEFAULT_SIZE: i32 = 32678;
+
+/// Similar to hyperloglog_trans(), except size is set to a default of 32,678
+#[pg_extern(immutable, parallel_safe,schema = "toolkit_experimental")]
+pub fn approx_count_distinct_trans(
+    state: Internal,
+    // TODO we want to use crate::raw::AnyElement but it doesn't work for some reason...
+    value: Option<AnyElement>,
+    fc: pg_sys::FunctionCallInfo,
+) -> Option<Internal> {
+    // let state: Internal = Internal::from_datum();
```

Comment:
Is the commented out code intentional? is it related to the TODO above?

Reply:
I kept this comment and the TODO since they were still both in the `hyperloglog_trans` function, I'm unsure if the two are related. I don't mind deleting them from this function